File: Python Excel Examples/WorksheetsApiDemo/deleteWorksheet.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "DeleteWorksheet.xlsx"
folder = "ExcelDocument";
storage = "";
sheetName = "Sheet1";
api.delete_worksheet(name, sheetName, folder=folder, storage=storage)

File: PHP Excel Examples/WorksheetsApiDemo/deleteWorksheet.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "DeleteWorksheet.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet2";
$result = $apiInstance->deleteWorksheet($name, $sheetName, $folder, $storage);